publish/publish_workflow.wdl

```diff
@@ -237,10 +237,10 @@ task prepGithub {
             --header "Authorization: token $(cat ~{githubTokenFile})" \
             --head \
             https://api.github.com/rate_limit \
-        | grep -i -F 'X-OAuth-Scopes:' > scopesHeader.txt
+        | grep -i -F 'X-OAuth-Scopes:' > scopesHeader.txt || true
```

Comment:
This is "similar but different" from what I removed in https://github.com/broadinstitute/cromwell/pull/6315. Assuming you tested this, 👍  and thanks

Reply:
> Assuming you tested this

Yep. Ran as part of the generated PR link in the comment above. That ` || true` is the difference between "wtf" and "oooooh". There are proper error messages generated below if that doesn't exit the script right there.